### ISSUE TITLE
enable span-tag to accept finc-Solr-Schema (JSON/ldj) as input

### ISSUE THREAD

**Issue by zazi** [miku/span#2]
(from my knowledge) currently, span-tag can consume Intermediate Schema (JSON/ldj). However, it might also be useful, if it can eat finc-Solr-Schema (JSON/ldj), because not all data source can be mapped to the Intermediate Schema.

**Comment by miku** [comment]
Yes, this is a currently a limitation. I started to make the filters a bit more independent in the  [sift](https://github.com/miku/span/tree/master/sift) subpackage, which only requires the certain small interfaces to be implemented to be able to use filters.

For example, a struct (which maps to JSON nicely) that implements [SerialNumberer](https://github.com/miku/span/blob/7e67d6cd1f0419b49f32c23763afcb2c872da704/sift/filter.go#L23-L26) should be able to use an [ISSN](https://github.com/miku/span/blob/7e67d6cd1f0419b49f32c23763afcb2c872da704/sift/issn.go#L21-L34) filter and so on.

What's left to do is to make SOLR schema implement these interfaces and then allow span-tag to map some name to some struct. I hope this will be done shortly.
